contributors/guide/pull-requests.md

```diff
@@ -109,6 +110,24 @@ easily see all criteria that are not being met and address them.
 
 That's the last step. Your pull request is now merged.
 
+## More About `Ok-To-Test`
+
+- For a Contributor, A `ok-to-test` label may just mean regular CI Testing to be done.
+- For the reviewer or the member, labelling the PR with `ok-to-test` it means a lot more:
+    + They need to take care if the PR is not a wastage of our `CI/CD` resources.
+    + Is the PR worth testing or does it needs more change before going through the `CI/CD` process ?
+    + Is the PR getting used to run malicious code to misuse our resources ?
+- An `ok-to-test` label may reduce the workload and smoothens the contributors experience as they can know if there is any failing test. If there is, you can fix the test and they don't have to wait for a long time to get a review from `maintainer/assignee`.
```

Comment:
I would actually remove this line or put a brief note at the top along the lines of:
The `ok-to-test` label is applied by org members to PRs from external contributors. It signals that the PR can be tested.